js/data/locale/ctype.json

```diff
@@ -476,12 +494,24 @@
             113823
         ]
     ],
+    "egyptian hieroglyph format controls": [
+        [
+            78896,
+            78911
+        ]
+    ],
     "elbasan": [
         [
             66816,
             66863
         ]
     ],
+    "elymaic": [
+        [
+            69600,
```

Comment:
How about 2 or 3 unit tests to verify these changes are working properly?

Reply:
Meaning 2 or 3 for the whole PR, not 2 or 3 per file. Just wanted to have a "smoke test" that it is working the way we expect.

Reply:
Yes, I've added some test cases.